### PR TITLE
perf: don't recalculate when eventsEnabled changes

### PR DESCRIPTION
**Pull request by jquense**
No need to rerun popper if events are enabled or disabled

### CODE REVIEW

File: src/Popper.js [hunk 1]
```diff
@@ -145,11 +149,11 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
 
     if (!referenceElement || !popperNode) return;
 
-    this.popperInstance = new PopperJS(
+    this.popperInstance = (new PopperJS(
       referenceElement,
       popperNode,
       this.getOptions()
-    );
+    ): any);
```
Comment (giladgray) [inline src/Popper.js]: why is this necessary? outdated types?

Comment (FezVrasta) [reply src/Popper.js]: If that's the case we really need to fix the types, otherwise we are missing most of the type coverage with this change.

Comment (jquense) [reply src/Popper.js]: I added the PR upstream already

Comment (FezVrasta) [reply src/Popper.js]: I merged and released it

Comment (jquense) [reply src/Popper.js]: Nice! I'll update the PR, thanks!

**Comment by FezVrasta** [comment]
@jquense I fixed the Flow types issues but we have some conflicts I'm not able to resolve. May you take a look please?

File: src/Popper.js [hunk 2]
```diff
@@ -182,6 +187,7 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
     // a size change it will result in a misaligned popper. So we schedule an update to be sure.
     if (prevState.placement !== this.state.placement) {
       this.scheduleUpdate();
+
```
Comment (FezVrasta) [inline src/Popper.js]: unneeded line

**Comment by jquense** [comment]
Any chance we could get a release @FezVrasta ?
